### PR TITLE
class for radiuses on both sides

### PR DESCRIPTION
**Pull request by csanner**
slight reorganization of the code for header/footer/middle to make it easier to add more styles in the future
added "-contained" to header, footer, and middle to have fully contained sections, resolving #25  #25

### CODE REVIEW

**Comment by csanner** [comment]
argh. I didn't actually mean to include my theme colors in this commit.
Feel free to remove or rename that part.

File: themes/lcars.yaml [hunk 1]
```diff
@@ -173,7 +173,6 @@
   app-header-edit-background-color: transparent
   switch-unchecked-track-color: var(--lcars-text-gray)
   switch-unchecked-button-color: var(--lcars-text-gray)
-  error-color: var(--lcars-error)
```
Comment (th3jesta) [inline themes/lcars.yaml:176]: Good catch.

File: themes/lcars.yaml [hunk 2]
```diff
@@ -660,7 +660,7 @@
         margin-left: -16px;
       }
     }
-    @media screen and (max-width: 375px) {
+    /*@media screen and (max-width: 375px) {
```
Comment (th3jesta) [inline themes/lcars.yaml:710]: I meant to remove this portion before release. It was an experimental fix for #38 before I figured out the actual cause. We can just remove this whole commented portion.

File: themes/lcars.yaml [hunk 3]
```diff
@@ -587,7 +587,7 @@
     ha-card:has(ha-gauge) {
       background: var(--lcars-card-top-color)
     }
-  /* Toolbar and dashboard modifications */
+  # Toolbar and dashboard modifications
```
Comment (th3jesta) [inline themes/lcars.yaml:637]: Yep, comments outside of the CSS follow YAML syntax, comments inside follow CSS standards.

File: themes/lcars.yaml [hunk 4]
```diff
@@ -490,6 +490,53 @@
       padding-bottom: 45px;
       color: black;
     }
+    ha-card.button-lozenge-right {
```
Comment (th3jesta) [inline themes/lcars.yaml:493]: THANK YOU SO MUCH

**Review by th3jesta — APPROVED** [review]

**Pull request merged**